modules/post/windows/gather/enum_patches.rb

```diff
@@ -34,32 +34,33 @@ def initialize(info={})
 
   def run
     unless load_extapi
-      print_error 'ExtAPI failed to load'
+      print_error("ExtAPI failed to load")
```

Comment:
So a few comments here, but in Ruby the common style is to use single quotes when not performing string interpolation and double quotes only when your performing string interpolation. Also, if the function only has a single parameter, its common to exclude the `(` and `)` characters.

On this basis I'm not sure this change is really needed.

Reply:
Fixed in next commit.